gateway/1.4/modules/ROOT/pages/_partials/task-config-logs-third-party.adoc

```diff
@@ -1,10 +1,21 @@
 //tag::intro1[]
-Anypoint Flex Gateway enables you to configure log output for consumption by third-party services. Logs can be delivered to any supported Fluent Bit {fluentbit-ver-var} output. You configure log output via a custom YAML file.
+Anypoint Flex Gateway enables you to deliver runtime and access logs to any supported Fluent Bit {fluentbit-ver-var} output type. You configure log output via a custom YAML file. Additionally, to view access logs, you must also apply a Message Logging policy.
```

Comment:
should the bit inside {} be in monospace font?

Reply:
Hanna, this is populated by an antora var.